airflow/cli/cli_parser.py

```diff
@@ -1381,6 +1388,18 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command('airflow.cli.commands.role_command.roles_create'),
         args=(ARG_ROLES, ARG_VERBOSE),
     ),
+    ActionCommand(
+        name='export',
+        help='Export roles from db to JSON file',
```

Comment:
```suggestion
        help='Export roles (without permissions) from db to JSON file',
```
or:
```suggestion
        help='Export role names from db to JSON file',
```

Or something similar, as I think most would expect the permissions to come with it?